### DISCUSSION TITLE
z/OS Open Tools and Open Mainframe Project

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#738] [Announcements]
Great news! z/OS Open Tools been [accepted](https://github.com/openmainframeproject/tac/issues/625#issuecomment-2012773319) into the [Open Mainframe Project (OMP)](https://openmainframeproject.org/)!

Joining the Open Mainframe Projects puts us alongside other leading IBM Z open source projects like Zowe, Ambitus, and CBTTape.

The Open Mainframe Project (affiliated with the Linux Foundation) will improve our credibility, raise awareness, attract more contributors through its mentorship program, and open up new opportunities for z/OS Open Tools!

This is a great step forward for z/OS Open Tools, and we're excited for the future!

**Comment by mgrossmann** [comment]
That is great news.

Congratulations!!!!

Von Gmail Mobile gesendet


Igor Todorovski ***@***.***> schrieb am Do. 21. März 2024 um
21:34:

**Comment by samdion1994** [comment]
Congratulations

**Comment by balhar-jakub** [comment]
It would be great to figure out, where we can as Zowe use the z/OS Open Tools.

**Comment by fernandofloresg** [comment]
Congratulations!!
